modules/vaos/app/services/vaos/systems_service.rb

```diff
@@ -64,6 +64,18 @@ def get_facility_limits(facility_id, type_of_care_id)
       end
     end
 
+    def get_facilities_limits(facility_ids, type_of_care_id)
+      with_monitoring do
+        url = "/var/VeteranAppointmentRequestService/v4/rest/direct-scheduling/patient/ICN/#{@user.icn}/request-limits"
```

Comment:
Can this url be moved into a method or even settings.yml?

Reply:
Good call, will move into a method.